lib/stripe/api.ex

```diff
@@ -16,7 +16,7 @@ defmodule Stripe.API do
   @typep http_failure :: {:error, term}
 
   @pool_name __MODULE__
-  @api_version "2018-08-23"
+  @api_version "2018-11-08"
```

Comment:
A few other updates that come along with this -> #439.  What do you think about backing out this change, merge 439, then get in your PR?

Reply:
@snewcomer sounds good, will do shortly 👍